### PR TITLE
Feat/civ 5835

### PR DESCRIPTION
**Pull request by lfazelyniaHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-5835

### Change description ###
Creating the screen for defendant repayment method when the claimant doesn't accept the proposed plan

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: codecept.conf.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ exports.config = {
     './e2e/tests/*_test.js',
```
Comment (lfazelyniaHMCTS) [inline codecept.conf.js:3]: Changes from Kenneth's story

File: e2e/api/steps_LRspecCui.js [hunk 2]
```diff
@@ -0,0 +1,352 @@
+const config = require('../config.js');
```
Comment (lfazelyniaHMCTS) [inline e2e/api/steps_LRspecCui.js:1]: Changes from Kenneths story

File: e2e/fixtures/events/defendantResponseSpecCui.js [hunk 3]
```diff
@@ -0,0 +1,516 @@
+const {listElement} = require('../../api/dataHelper');
```
Comment (lfazelyniaHMCTS) [inline e2e/fixtures/events/defendantResponseSpecCui.js:1]: changes from Kenneths story

File: e2e/run-api-ff-enabled-functional-tests.sh [hunk 4]
```diff
@@ -11,3 +11,6 @@ yarn test:api-spec
```
Comment (lfazelyniaHMCTS) [inline e2e/run-api-ff-enabled-functional-tests.sh:11]: Changes from Kenneths story

File: e2e/run-spec-functional-tests.sh [hunk 5]
```diff
@@ -5,4 +5,7 @@ echo "Run Api Functional tests for api-spec"
 yarn test:api-spec
```
Comment (lfazelyniaHMCTS) [inline e2e/run-spec-functional-tests.sh:5]: Changes from Kenneths story

File: e2e/tests/api_tests/lrspec_cui/api_1v1_spec_cui_test.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-unused-vars */
```
Comment (lfazelyniaHMCTS) [inline e2e/tests/api_tests/lrspec_cui/api_1v1_spec_cui_test.js:1]: Changes from Kenneths story

File: package.json [hunk 7]
```diff
@@ -28,6 +28,7 @@
     "test:api-spec-1v2": "MOCHAWESOME_REPORTFILENAME=api-spec npx codeceptjs run --grep @api-spec-1v2 --reporter mocha-multi --verbose",
     "test:api-spec-1v1": "MOCHAWESOME_REPORTFILENAME=api-spec npx codeceptjs run --grep @api-spec-1v1 --reporter mocha-multi --verbose",
     "test:api-spec-small": "MOCHAWESOME_REPORTFILENAME=api-spec npx codeceptjs run --grep @api-spec-small --reporter mocha-multi --verbose",
+    "test:api-spec-cui": "MOCHAWESOME_REPORTFILENAME=api-spec-cui npx codeceptjs run --grep @api-spec-cui --reporter mocha-multi --verbose",
```
Comment (lfazelyniaHMCTS) [inline package.json:31]: Changes from Kenneths story

**Review by laperla11 — APPROVED** [review]

**Review by kenneth-hmcts — APPROVED** [review]

**Review by portizdev — APPROVED** [review]

**Pull request closed without merging**